### DISCUSSION TITLE
Prompt Template + Buffer Memory (conversational agent against a specific prompt)

### DISCUSSION THREAD

**Discussion by ryandeschamps** [FlowiseAI/Flowise#150] [Q&A] [answered]
Hi,

I have been unable to figure out how to create a flow that includes the prompt template and a conversational (memory) aspect.  Specifically, I am trying to create a flow that creates an agent (through the prompt template), but I want the user to be able to continue to have a conversation.

Does the framework support this?

Thanks!

**Comment by HenryHengZJ** [comment, marked as answer]
Unfortunately we don't support dynamic creation of components like agent/chain from prompt
